core/tauri-macros/src/lib.rs

```diff
@@ -151,3 +154,35 @@ pub fn do_menu_item(input: TokenStream) -> TokenStream {
   let tokens = parse_macro_input!(input as menu::DoMenuItemInput);
   menu::do_menu_item(tokens).into()
 }
+
+/// Convert a .png or .ico icon to an Image
+/// for things like `tauri::tray::TrayIconBuilder` to consume,
+/// relative paths are resolved from `CARGO_MANIFEST_DIR`, not current file
```

Comment:
should it be relative to current file to match `include_str` and similar macros behavior?

Reply:
How to do that? I found it being possible in nightly rust, but it's not stable yet

> https://stackoverflow.com/a/65813587/16993372

Reply:
I see, I thought I saw another macro doing that but I remember now that they were using `CARGO_MANIFEST_DIR` as well